page/javascript-101/testing-type.md

```diff
@@ -44,4 +44,4 @@ if ( Object.prototype.toString.call( myArray ) === "[object Array]" ) {
 }
 ```
 
-jQuery also offers utility methods to help determine the type of an arbitrary value.
+jQuery also offers [utility methods](../using-jquery-core/utility-methods.md#testing-type) to help determine the type of an arbitrary value.
```

Comment:
you do need a leading /

this will work /using-jquery-core/utility-methods#testing-type